app/assets/main/react_components/dashboard/climate_actions/manage_actions/kanban/KanbanActionItem.jsx

```diff
@@ -64,12 +63,11 @@ const KanbanActionItem = ({
       {(provided, snapshot) => {
         return (
           <div
-            className={`border border-gray-tint-2 rounded-lg space-y-3 ${
-              collapsed ? "w-24" : "t:w-60 d:w-80 d-lg:w-96"
+            className={`border border-gray-tint-2 rounded-lg shadow-lg p-0 space-y-3 pt-0 w-80 ${
+              collapsed ? "d:w-24" : "t:w-80 d:w-80 d-lg:w-80"
```

Comment:
Is this on purpose or by mistake? If its 80 for all sizes we can skip the two last ones. But I think the breakpoints are quite good? then remove the w-80 also before the "collapsed ?..."